README.md

```diff
@@ -1 +1,169 @@
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>🖥️ Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/REACT/">React</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+</details>
+
+
+<!-- FEATURES -->
+
+### 🔑 Key Features <a name="key-features"></a>
+
+- [ ] **Using react**
+- [ ] **Using routers**
+- [ ]  **Linter**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+To run this project, you need the following:
+
+-  Web browser
+-  Git
+-  Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+git clone https://github.com/mahdinoori2000/bookstore.git
+```
+
+### Install
+
+To install all dependencies, run the following command:
+
+```bash
+npm install
+```
+
+- To track CSS linter errors, run:
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+- To track JavaScript linter errors, run:
+```bash
+npx eslint "**/*.{js,jsx}"
+```
+
+### Build
+
+To build the project, run the following command:
+
+```bash
+npm run build
+```
+
+### Webpack Development Server
+
+To run the webpack development server, execute the following command:
+
+```bash
+npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👤 Author <a name="author"></a>
+
+👤 Mahdi Noori
+
+- GitHub: [@mahdinoori2000](https://github.com/mahdinoori2000)
+- Twitter: [@mahdinoori2000](https://twitter.com/mahdinoori2000)
+- LinkedIn: [@mahdinoori2000](https://www.linkedin.com/in/mahdi-noori-4b4370270/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔮 Future Features <a name="future-features"></a>
+
+- [ ]  **Functionality to add book with author**
+- [ ] **Functionality to remove book**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions to Math Magicians are welcome! 
+
+Feel free to check the [issues page](https://github.com/mahdinoori2000/bookstore/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SHOW YOUR SUPPORT -->
+
+## ⭐️ Show Your Support <a name="show-your-support"></a>
+
+If you like this project, give it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgments"></a>
+
+I would like to thank Microverse for providing guidance of the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is licensed under the [MIT License](./LICENSE).
```

Comment:
- [ ] This link is broken. kindly consider fixing this link as having broken links in our readme makes it unprofessional.